.github/workflows/verify.yml

```diff
@@ -64,10 +64,10 @@ jobs:
       fail-fast: true
       matrix:
         ruby:
-          - '2.7'
```

Comment:
Ruby 2.7 eol was 3 months ago; May as well swap in the new for the old

Reply:
It might still be supported by vendors, like in [Debian (bullseye) stable]( https://wiki.debian.org/Ruby ). If it doesn't increase support burden, it would be nice to keep it :)

Reply:
I'm on the fence on this as I think it does increase the support burden but last I checked I don't think we believed it was big enough yet to warrant a full update just yet, but were were planning to get around to it soon. With all the recent work to get up to par with recent 3.X additions though I am leaning towards just dropping 2.7 support since its increasingly likely newer 3.X additions might end up breaking 2.7 compatibility as Ruby starts to venture into more dramatic changes to its code base (at least in my eyes that's what I'm seeing anyway).

Reply:
Will open the 2.7 removal as a separate PR